src/main/resources/synthea.properties

```diff
@@ -14,9 +14,12 @@ exporter.fhir.transaction_bundle = false
 exporter.hospital.fhir.export = true
 exporter.hospital.fhir_dstu2.export = false
 exporter.csv.export = false
+
 exporter.text.export = false
+exporter.text.export2 = false
```

Comment:
Could you use a more descriptive name than export2? It's not immediately clear what this means. Maybe `exporter.text.per_encounter_export` or something like that.